README.md

```diff
@@ -44,8 +51,8 @@ Current Canopy-App components:
     Namelist Option : `file_vars`  Full name of input file (Supports either text or NetCDF format with following formats:
                                                             `.txt`, `.nc`, `.ncf`, or `.nc4`)
 
-- See example file inputs for variables and format (`input_variables.txt`, `input_variables_1D.nc`, or `input_variables_2D.nc`).
-- Canopy-App assumes the NetCDF input files are in CF-Convention; recommend using double or float for real variables.
+- See example file inputs for variables and format (`input_variables.txt` or `testf000.nc`).
```

Comment:
Is there a significance to the `000`? If so, should it be explained?

Reply:
@zmoon Good point.  Added some explanation.